lib/standards/tests/textMustBeStyledWithUnitsThatAreResizableInAllBrowsers.js

```diff
@@ -56,7 +56,7 @@ function getElementFontSizes (el) {
 function fontSizesIncludeUnit (sizes, unit) {
   for (var sizeIndex in sizes) {
     var size = sizes[sizeIndex]
-    if (size.indexOf(unit) > -1) {
+    if (new RegExp('[0-9.]+' + unit).test(size)) {
```

Comment:
Doesn't the `.` need to be escaped? `\.`
Otherwise it matches any character.

Reply:
It doesn't need to be escaped when in `[]`

Reply:
```js
new RegExp('[0-9.]+in').test('5.5in')
true
new RegExp('[0-9.]+in').test('5in')
true
new RegExp('[0-9.]+in').test('gin')
false
```

Reply:
Ok, then this is good to merge!